### PR TITLE
Reform: End Entrenched Poverty, Scott Budow

### LINKED ISSUE #3491: Reform: End Entrenched Poverty, Scott Budow
Scott Budow has recently published a [proposal](https://www.scottbudow.com/issues?id=f5b39b46-f3ca-4fc8-a546-9229dbfbd487) named "End Entrenched Poverty" which introduces the idea of a Cash Assistance program for children and families below poverty line (2x below poverty line in some cases)

### PR DESCRIPTION
**Pull request by PavelMakarchuk**
Fixes #3491

copilot:all

### CODE REVIEW

**Review by MaxGhenis — CHANGES_REQUESTED** [review]

File: policyengine_us/reforms/scott_budow/end_entrenched_poverty/end_entrenched_poverty.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+                parameters(period).gov.states.ny.tax.income.credits.refundable,
+            )
+            end_entrenched_poverty = tax_unit(
+                "end_entrenched_poverty_credit", period
+            )
+            return old_credits + end_entrenched_poverty
+
+    class reform(Reform):
+        def apply(self):
+            self.update_variable(end_entrenched_poverty_credit)
```
Comment (PavelMakarchuk) [inline policyengine_us/reforms/scott_budow/end_entrenched_poverty/end_entrenched_poverty.py:49]:
```suggestion
            self.create_variable(end_entrenched_poverty_credit)
```

File: policyengine_us/parameters/gov/contrib/scott_budow/end_entrenched_poverty/child.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+description: New York provides cash assistance for each child living under the poverty line of this amount, under Scott Budow's end entrenched poverty proposal.
+metadata:
+  type: single_amount
+  threshold_unit: currency-USD
+  amount_unit: /1
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/contrib/scott_budow/end_entrenched_poverty/child.yaml:5]:
```suggestion
  amount_unit: currency-USD
```

File: policyengine_us/parameters/gov/contrib/scott_budow/end_entrenched_poverty/child.yaml [hunk 3]
```diff
@@ -0,0 +1,19 @@
+description: New York provides cash assistance for each child living under the poverty line of this amount, under Scott Budow's end entrenched poverty proposal.
+metadata:
+  type: single_amount
+  threshold_unit: currency-USD
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/contrib/scott_budow/end_entrenched_poverty/child.yaml:4]:
```suggestion
  threshold_unit: /1
```

**Comment by PavelMakarchuk** [comment]
Duplicate of #3968

**Pull request closed without merging**